YoutubeKit/API/Models/Thumbnail.swift

```diff
@@ -23,15 +23,11 @@ extension Thumbnails {
         public let high: Default
         public let `default`: Default
         public let medium: Default
-        public let standard: Default
```

Comment:
Is this diff need for this PR?
Dup: https://github.com/rinov/YoutubeKit/pull/15

But ok 👌